Makefile.am

```diff
@@ -24,6 +24,7 @@ bin_PROGRAMS = memtier_benchmark
 memtier_benchmark_CPPFLAGS = $(LIBEVENT_CFLAGS)
 memtier_benchmark_SOURCES = \
 	memtier_benchmark.cpp memtier_benchmark.h \
+	run_stats.cpp run_stats.h \
```

Comment:
@YaacovHazan I think it would be a good idea to separate the stats code reorganization from the MOVED commit.

Reply:
ok, I will move it back